src/lib/tooltip/tooltip.ts

```diff
@@ -392,6 +427,113 @@ export class MdTooltip implements OnDestroy {
+   */
+  private _unregisterA11yMessage(message: string) {
+    if (!this._platform.isBrowser || !this.message) { return; }
+
+    const a11yMessageElement = a11yMessages.get(message)!;
+    a11yMessageElement.count--;
+
+    // Remove the a11y message if this was its last unique instance.
+    if (a11yMessageElement.count == 0) {
+      this._deleteA11yMessageElement(message);
+    }
+
+    // If the global messages container no longer has any children, remove it.
+    if (!this._getA11yMessagesContainer()!.childNodes.length) {
+      this._renderer.removeChild(document.body, this._getA11yMessagesContainer());
+    }
+
+  }
+
+  /**
+   * Creates a new element in the visually hidden a11y message container element with the message
+   * as its content.
+   */
+  private _createA11yMessageElement(message: string) {
+    if (!this._platform.isBrowser) { return; }
+
+    // Create a visually hidden container for the accessibility messages if one does not yet exist.
+    if (!this._getA11yMessagesContainer()) {
+      this._createA11yMessagesContainer();
+    }
+
+    const messageElement = this._renderer.createElement('div');
+    messageElement.id = `md-tooltip-message-${latestA11yMessageId++}`;
+    messageElement.textContent = message;
+
+    this._getA11yMessagesContainer()!.appendChild(messageElement);
+    a11yMessages.set(message, {element: messageElement, count: 1});
+
+    return messageElement;
+  }
+
+  /** Deletes the a11y message element from the global a11y tooltip messages container. */
+  private _deleteA11yMessageElement(message: string) {
+    if (!this._platform.isBrowser) { return; }
+
+    const a11yMessage = a11yMessages.get(message)!;
+    this._renderer.removeChild(this._getA11yMessagesContainer()!, a11yMessage.element);
+    a11yMessages.delete(message);
+  }
+
+  /** Returns the trigger's aria-describedby attribute. */
+  private _getAriaDescribedby(): string {
+    return this._elementRef.nativeElement.getAttribute('aria-describedby');
+  }
+
+  /** Sets the trigger's aria-describedby attribute to the tooltip message element. */
+  private _setAriaDescribedBy() {
+    // Return if an aria-describedby already exists and it is not referencing a tooltip message.
+    const ariaDescribedBy = this._elementRef.nativeElement.getAttribute('aria-describedby');
+    if (ariaDescribedBy && ariaDescribedBy.indexOf('md-tooltip-message') == -1) { return; }
+
+    if (this.message) {
+      const tooltipMessageElementId = a11yMessages.get(this.message)!.element.id;
+      this._renderer.setAttribute(
+          this._elementRef.nativeElement, 'aria-describedby', tooltipMessageElementId);
+    } else {
+      this._renderer.setAttribute(this._elementRef.nativeElement, 'aria-describedby', '');
+    }
+  }
+
+  /** Creates the global container for all tooltip a11y messages. */
+  private _createA11yMessagesContainer() {
+    const a11yMessagesContainer = this._renderer.createElement('div');
```

Comment:
This needs `aria-hidden` as well. Otherwise screen-reader users will run into this content at the end of the document without context